style.css

```diff
@@ -19,62 +84,184 @@
     width: 0;
 }
 
-.products > h1 {
+.products>h1 {
     padding-left: 1rem;
 }
 
 button {
     cursor: pointer;
 }
 
-form > div {
+form>div {
     margin: 2rem 0;
 }
 
-form  > div > div > div label{
+form>div>div>div label {
     font-size: 0.1rem;
 }
 
-form > div > h2,
-.products h1 {
+form>div>h2,
+.products>h1 {
     font-size: 1rem;
     margin-bottom: 0.6rem;
 }
 
-.price > div {
+.price>div {
     margin-bottom: -0.6rem;
     font-size: 0.9rem;
     color: var(--text-light);
     width: 12rem;
 }
 
-form .price-label{
+form .price-label {
     margin-top: 0.6rem;
     max-width: 100%;
 }
 
-.price > .range-input,
-.brand > div > div,
-.rating > div,
-.sort > div {
+.price>.range-input,
+.brand>div>div,
+.rating>div,
+.sort>div {
     width: 100%;
     margin-left: 0;
 }
 
-form > .filter > h2,
-form > div > button {
+form>.filter>h2,
+form>div>button {
     padding: 0.3rem auto;
     font-size: 1rem;
     border: none;
     background: none;
     color: var(--text-dark);
 }
 
-form > div > button{
+form>div>button {
     text-decoration: underline;
     align-self: flex-start;
 }
 
 .modal-active-shadow {
     box-shadow: 0 0 0 300rem rgba(0, 0, 0, 0.7);
```

Comment:
here too.

Reply:
Here I am playing with the alpha channel, that's why didn't use variables.